src/js/shared/dates.ts

```diff
@@ -23,19 +24,19 @@ export function getEarliestGraduationDateToAccept() {
  * Returns the datetime at which the hacking period begins.
  */
 export function getHackingPeriodStart() {
```

Comment:
@henrymercer is this function (and the others below) supposed not to have an annotation?

Reply:
They're optional, but you're right I'll add them for consistency.